main.py

```diff
@@ -295,6 +299,42 @@ def eval_worker(gpu, ngpu_per_node, args, res_queue=None):
              world_size=args.world_size, distributed=args.distributed,
              best_metric=best_metric, res_queue=res_queue, pad=args.pad)
 
+def df_to_bigwig(intervals, args, batch_data, outputfile):
```

Comment:
let's move this function into `bigwigio` for now? And a task to write unit tests for it so we have don't end up breaking it

Reply:
done.